lib/monitoring/aws-rds/RdsClusterMonitoring.ts

```diff
@@ -147,15 +166,106 @@ export class RdsClusterMonitoring extends Monitoring {
       this.addAlarm(createdAlarm);
     }
 
+    for (const disambiguator in props.addSelectLatencyAlarm) {
+      const alarmProps = props.addSelectLatencyAlarm[disambiguator];
+      const createdAlarm = this.latencyAlarmFactory.addLatencyAlarm(
+        this.selectLatencyMetric,
+        LatencyType.P90,
+        alarmProps,
+        disambiguator,
+        "Select",
+      );
+      this.latencyAnnotations.push(createdAlarm.annotation);
```

Comment:
Would the datapoints for this end up being much higher than things like insert/update/delete and making the graph kind of hard to read?